TrustModel.md

```diff
@@ -0,0 +1,113 @@
+# Trust Model for Confidential Containers
+A clear definition of trust for the confidential containers project is needed to ensure the 
+components and architecture deliver the security principles expected for cloud native 
+confidential computing. It provides the solid foundations and unifying security principles 
+against which we can assess architecture and implementation ideas and discussions.
+
+ Trust modeling can be defined[^5] as :
+
+    A trust model identifies the specific mechanisms that are necessary to respond to a specific 
+    threat profile.
+
+    A trust model must include implicit or explicit validation of an entity's identity or the 
+    characteristics necessary for a particular event or transaction to occur.
+
+ The trust model also helps determine the location and direction of the trust boundaries where a 
+[trust boundary](https://en.wikipedia.org/wiki/Trust_boundary) describes a location where 
+ program data or execution changes its level of "trust", or where two principals with different 
+ capabilities exchange data or commands. Specific to Confidential Containers is the trust 
+ boundary that corresponds to the boundary of the Trusted Execution Environment (TEE). Generally, 
+ the TEE side of the trust boundary will be hardened to prevent the violation of the trust 
+ boundary.
+
+In order to describe and understand particular threats we need to establish trust boundaries and 
+trust models relating to the key aspects, components and actors involved in Cloud Native 
+Confidential Computing. We explore trust using different orthogonal ways of considering cloud 
+native approaches when they use an underlying Trusted Execution Environment technology and 
+identifying where there may be considerations to preserve the value of using a TEE.
+
+<!--
+   Follow on pull requests will list here subsequent sections corresponding to orthogonal ways 
+of considering cloud native approaches
+Specifically including
+   - Personas
+   - Lifecycle
+   - Trusted Execution Boundary Location
+   - Secured v unsecured TEE Base Image
+-->
```

Comment:
I only put this in so that people can see what the follow on PR's will contain, happy to just remove this from this PR if that is better?

Reply:
I found it helpful to see this list since we split it out of #16.
In that earlier PR you referred to [guest] `boot image` in a few places. I take that to mean what you here call `base image`?
Also note that's only relevant to the pod/vm approach.

Reply:
Yes I'll leave the comment out list there then so everyone can be clear what is coming

Reply:
If we plan to land this PR, I think the list should be moved to a GH issue (now) to allow more visible tracking of outstanding items, and easier and quicker updates to that list.

Reply:
The list does currently exist in this issue -> https://github.com/confidential-containers/documentation/issues/14
So happy to remove this and rely on issue

Reply:
I have create two new issues to give a fresh basis for the continued discussion
#28 covers a list of Trust model documentation to be completed
#29 covers a list of Threats Vectors/Profiles we should document (should use concepts and terminology agreed in model documentation)